### PR TITLE
update release notes for compose version v2.22.0

### PR DESCRIPTION
**Pull request by glours**
### Proposed changes
Update Compose release notes for version `v2.22.0`

### CODE REVIEW

File: content/compose/release-notes.md [hunk 1]
```diff
@@ -11,6 +11,28 @@ aliases:
 
 For more detailed information, see the [release notes in the Compose repo](https://github.com/docker/compose/releases/).
 
+## 2.22.0
+{{< release-date date="2023-09-21" >}}
+
+> **Note**
+>
+> `watch` command is now GA and can be directly used from the root command `docker compose watch`
```
Comment (usha-mandya) [inline content/compose/release-notes.md:19]: @glours Thanks for the PR. Could you add a link to the Compose watch docs/CLI reference here?

File: content/compose/release-notes.md [hunk 2]
```diff
@@ -11,6 +11,28 @@ aliases:
...
+
+### Update
+- Dependencies upgrade: bump golang to 1.21.1
+- Dependencies upgrade: bump compose-go to v1.19.0
+- Dependencies upgrade: bump buildkit to v0.12.2
+
+### Bug fixes and enhancements
+- Added experimental support of `publish` command
+- `watch` now build and launch the project at start
+- Add `policy` option to the `--pull` flag
```
Comment (usha-mandya) [inline content/compose/release-notes.md:29]:
```suggestion
- Added `policy` option to the `--pull` flag
```

File: content/compose/release-notes.md [hunk 3]
```diff
@@ -11,6 +11,28 @@ aliases:
...
+- Fixed various race/deadlock conditions for `up` command on exit
```
Comment (usha-mandya) [inline content/compose/release-notes.md:30]:
```suggestion
- Fixed various race and deadlock conditions for `up` command on exit.
```

File: content/compose/release-notes.md [hunk 4]
```diff
@@ -11,6 +11,28 @@ aliases:
...
+- Fixed for multi-plaform issues on build
```
Comment (usha-mandya) [inline content/compose/release-notes.md:31]:
```suggestion
- Fixed multi-platform issues on build.
```

File: content/compose/release-notes.md [hunk 5]
```diff
@@ -11,6 +11,28 @@ aliases:
...
+- Enabled services explicitly requested even if their `profiles` aren't activated
```
Comment (usha-mandya) [inline content/compose/release-notes.md:32]:
```suggestion
- Enabled services that are explicitly requested even when their `profiles` aren't activated.
```

File: content/compose/release-notes.md [hunk 6]
```diff
@@ -11,6 +11,28 @@ aliases:
...
+- Fixed `config` bug when declared `env_file` are missing
```
Comment (usha-mandya) [inline content/compose/release-notes.md:33]:
```suggestion
- Fixed  a `config` issue when the declared `env_file` is missing.
```

File: content/compose/release-notes.md [hunk 7]
```diff
@@ -11,6 +11,28 @@ aliases:
...
+- Passed BuildOptions to `up` and `run` commands
```
Comment (usha-mandya) [inline content/compose/release-notes.md:34]:
```suggestion
- Passed BuildOptions to `up` and `run` commands.
```

File: content/compose/release-notes.md [hunk 8]
```diff
@@ -11,6 +11,28 @@ aliases:
...
+> `watch` command is now GA and can be directly used from the root command `docker compose watch`
+
+### Update
+- Dependencies upgrade: bump golang to 1.21.1
+- Dependencies upgrade: bump compose-go to v1.19.0
+- Dependencies upgrade: bump buildkit to v0.12.2
+
+### Bug fixes and enhancements
+- Added experimental support of `publish` command
+- `watch` now build and launch the project at start
```
Comment (usha-mandya) [inline content/compose/release-notes.md:28]:
```suggestion
- The command `watch` now builds and launches the project during startup.
```

File: content/compose/release-notes.md [hunk 9]
```diff
@@ -11,6 +11,28 @@ aliases:
...
+>
+> `watch` command is now GA and can be directly used from the root command `docker compose watch`
+
+### Update
+- Dependencies upgrade: bump golang to 1.21.1
+- Dependencies upgrade: bump compose-go to v1.19.0
+- Dependencies upgrade: bump buildkit to v0.12.2
+
+### Bug fixes and enhancements
+- Added experimental support of `publish` command
```
Comment (usha-mandya) [inline content/compose/release-notes.md:27]:
```suggestion
- Added experimental support for the `publish` command.
```

File: content/compose/release-notes.md (hunk 1 above)
Comment (glours) [reply content/compose/release-notes.md:19]: Done!

File: content/compose/release-notes.md [hunk 10]
```diff
@@ -11,6 +11,29 @@ aliases:
 
 For more detailed information, see the [release notes in the Compose repo](https://github.com/docker/compose/releases/).
 
+## 2.22.0
+{{< release-date date="2023-09-21" >}}
+
+> **Note**
+>
+> `watch` command is now GA and can be directly used from the root command `docker compose watch`
```
Comment (usha-mandya) [inline content/compose/release-notes.md:19]:
```suggestion
> The `watch` command is now generally available (GA). You can directly use it from the root command `docker compose watch`.
```

File: content/compose/release-notes.md [hunk 11]
```diff
@@ -11,6 +11,29 @@ aliases:
...
+> [Check official documentation of the watch feature](./file-watch.md) for more details
```
Comment (usha-mandya) [inline content/compose/release-notes.md:20]:
```suggestion
> For more information, see [File watch](./file-watch.md).
```

**Review by usha-mandya — APPROVED** [review]

**Pull request merged**